runner-config.js

```diff
@@ -20,6 +20,16 @@ const webRunTests = require('./test/tasks/webRunTests');
 let logServerPort;
 let staticPort;
 
+const jsFilesFilter = item => path.extname(item.path) === '.js'
+let shimSpecificTests = walk(path.join(__dirname, 'test', 'suites'), {
+    filter: jsFilesFilter,
+    nodir: true
+});
+let commonTests = walk(path.join(__dirname, 'node_modules/kinvey-js-sdk/test/commonTests'), {
```

Comment:
It's better to provide each folder as a separate argment to `path.join` instead of hardcoding slashes for better cross-platform path support